internal/mqini/mqini.go

```diff
@@ -76,3 +88,168 @@ func GetErrorLogDirectory(qm *QueueManager) string {
 	}
 	return filepath.Join(qm.Prefix, "qmgrs", qm.Directory, "errors")
 }
+
+//Update qm.ini file with the user supplied stanzas.
+func AddStanzas(qmname string) error {
+
+	qm, err := GetQueueManager(qmname)
```

Comment:
Formatting looks off in a number of places.  Code needs to be passed through "go fmt" to format it the right way.  Most editors do this, but perhaps not the one you're using.

Reply:
I have ran the latest commit with gofmt and formatted the code.